templates/workflow/workflow.tpl

```diff
@@ -33,15 +33,7 @@
 			>
 				{translate key="common.declined"}
 			</badge>
-			<span class="pkpWorkflow__identificationId">{{ submission.id }}</span>
-			<span class="pkpWorkflow__identificationDivider">/</span>
-			<span class="pkpWorkflow__identificationAuthor">
-				{{ currentPublication.authorsStringShort }}
-			</span>
-			<span class="pkpWorkflow__identificationDivider">/</span>
-			<span class="pkpWorkflow__identificationTitle">
-				{{ localizeSubmission(currentPublication.fullTitle, currentPublication.locale) }}
-			</span>
+			{include file="core:workflow/submissionIdentification.tpl" submission=$submission currentPublication=$currentPublication}
```

Comment:
The template variables don't need to be passed down here because the `{include}` will inherit the template variables. You only need to pass in a template variable when the included template uses a different variable name. For example:

```tpl
// title.tpl
<h1>{$publication->getFullTitle()}</h1>
```

```tpl
// parent template
// notice how one variable ($currentPublication) is passed to the included template as another variable ($publication)
{include file="title.tpl" publication=$currentPublication}
```

Also, remove the `core:` prefix from here and other places. That will prevent an application from overriding the template if it wants to.